source/arroost/components/tunnel.js

```diff
@@ -227,7 +272,7 @@ const noop = () => {}
 /** @type {TunnelMap} */
 const TUNNELS = {
 	fired: ({ id }) => {
-		const tunnel = Tunnel.tunnels.get(id)
+		const tunnel = Tunnel.get(id)
```

Comment:
replaced the tunnels map methods with some sugar so we can do any side effects we want